### PR TITLE
Feature/1 Initial setup

### PR DESCRIPTION
**Pull request by krissy**
Addresses feature: #1 

Initial setup, including dev tools and base API + tests.
### Tasks
- [x] Set up base app with default route, config management
- [x] Set up test suite (use Mocha/Chai/Supertest)
- [x] Set up task runs via [Gulp](gulpjs.com)
- [x] Set up CI via [Travis](https://travis-ci.org)
- [x] Add gitignore
- [x] Update README

### CODE REVIEW

File: config.js.example [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/**
```
Comment (tanyapowell) [inline config.js.example:1]: Was this file committed by mistake?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,451 @@
...
+    "createdAt": "2016-01-21 10:45:01",
+    "updatedAt": "2016-01-21 10:45:01",
+    "updatedByUser": "tanya"
+  }
+}
+
+```
+
+
+### Volunteers
```
Comment (tanyapowell) [inline README.md:290]: Are you storing volunteer details on your DB until the volunteer portal is ready? I.e. is this temporary?

File: config.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    'server': {
+      'port': process.env.PORT || 8080,
+      'host': 'http://localhost',
+      'apiPath': '/api'
+    },
+    'database': {
+      'url': 'mongodb://localhost/whfnp-content-test'
+    }
+  },
+  'production': {
```
Comment (tanyapowell) [inline config.js:23]: Mongolab offers a free 500MB tier. Could be worth using for production and possibly a test DB

File: gulp/test.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+ */
+gulp.task('env:test', function() {
+  process.env.NODE_ENV = 'test';
+});
+
+/**
+ * Run Jasmine tests
+ */
+gulp.task('runJasmine', ['env:test'], function() {
+  return gulp.src(['test/*spec.js'])
```
Comment (tanyapowell) [inline gulp/test.js]: Is `'test/*spec.js'` any different from `'test/*.js'`?

File: config.js.example (hunk 1 above)
Comment (krissy) [inline config.js.example:1]: Ideally the actual config.js will be added to .gitignore and we'll only push up the .example file, but Travis isn't playing nicely without this file committed to the repo. Will sort that bit out in a later PR.

File: README.md (hunk 2 above)
Comment (krissy) [inline README.md:290]: The board member and volunteer info here is limited to what we need to output in website content, which is why it made sense to segregate them in this content API. The volunteer-focussed API will contain additional details more in line with volunteer management in the Volunteer/Project portal. Discussed with Vinita/Dionne et al so we're all on the same page about that. Also, that it's highly likely we can integrate the Portal API in the future so some of these details _could_ come from there eventually.

File: gulp/test.js (hunk 4 above)
Comment (krissy) [inline gulp/test.js]: SettingJasmine is to run anything post-fixed with `spec,js`, as there will eventually be some setup files in the test folder that aren't necessarily needed for each test run.

File: config.js (hunk 3 above)
Comment (krissy) [inline config.js:23]: :dancers:

**Comment by krissy** [comment]
Going to merge this in to `dev` and continue with the pending issues. Please feel free to add comments following the latest commits / test framework switch and I'll address in a future PR.

**Pull request merged**